thirdeye/thirdeye-frontend/app/pods/components/entity-filter/component.js

```diff
@@ -110,42 +106,42 @@ export default Component.extend({
    * Array containing the running list of all user-selected filters
    * @type {Array}
    */
+  // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
You can do this to avoid having to bypass eslint here-

```suggestion
 import { A as EmberArray } from '@ember/array';
 alertFilters: EmberArray()
```